### PR TITLE
Change `sub_list`data type

### PR DESCRIPTION
**Pull request by alvin1221**

### CODE REVIEW

**Review by JaylinYu — COMMENTED** [review]
why?

**Review by wanghaEMQ — APPROVED** [review]

File: include/nng/supplemental/nanolib/conf.h [hunk 1]
```diff
@@ -253,7 +253,7 @@ struct conf_bridge_node {
 	char        *password;
 	char       **forwards;
 	uint64_t     parallel;
-	topics      *sub_list;
+	topics     **sub_list;
```
[the remark this answers is not in the document]
Comment (alvin1221) [reply include/nng/supplemental/nanolib/conf.h:256]: See https://github.com/emqx/nanomq/pull/1219

[the remark this answers is not in the document]
Comment (wanghaEMQ) [reply include/nng/supplemental/nanolib/conf.h:256]: Yes. I got it

**Review by lee-emqx — APPROVED** [review]

**Review by JaylinYu — APPROVED** [review]

**Comment by JaylinYu** [comment]
Annotation!!!!!!

**Pull request merged**